### ISSUE TITLE
Question about estimating the noise transition matrix?

### ISSUE THREAD

**Issue by rosefun** [Sunarker/Safeguarded-Dynamic-Label-Regression-for-Noisy-Supervision#4]
After reading the paper, I'm still confused about estimating the noise transition matrix.
Do you still use a Softmax layer to estimate it or something else?
Maybe I don't read it carefully. Hopefully for your reply.

**Comment by Sunarker** [comment]
Sorry for late reply. Firstly, I directly train the classifier on the noisy training dataset, and then, estimate the noise transition via the Softmax layer like the Eq. (9) in the paper. The codes may be slightly different due to unremoval trials. Just trying the way in the paper, a good result will achieve.

**Comment by rosefun** [comment]
One more question, how do you initialize the confuse matrix N?

**Comment by Sunarker** [comment]
For each training sample, sampling the label based on Eq. (6), and then compute the confuse matrix. Note that, if you use the noise transition matrix initialization, replace the conditional transition in Eq. (6) with the pre-estimated noise transition.
